### DISCUSSION TITLE
Commit Reports on different branches

### DISCUSSION THREAD

**Discussion by ghost** [TheAlbertDev/MASB#45] [Q&A] [answered]
Hello again Albert. 

I'm trying tu upload the REPORTS in the arduino and stm32cube folders. 

In the develop/A branch, I `add` and `commit` the arduino's REPORT and the terminal "frozes" (I cannot write and I'm forced to close the terminal) with this message: 

![commit arduino A](https://user-images.githubusercontent.com/78802420/108633921-44416a00-7477-11eb-8f4d-0f0daea5fa6d.JPG)

On the other hand, when I `add` and `commit` the stm's REPORT in the develop/B-branch. An analogous message appears too: 

![commit stm B](https://user-images.githubusercontent.com/78802420/108633941-58856700-7477-11eb-814a-0e3ee8e99ab5.JPG)

No report appears in the GitHub website. After running `git status` (next image) I'm afraid I might have comitted both reports in both branches. Is that possible? If positive, can I delete this commit or start over?

![status](https://user-images.githubusercontent.com/78802420/108634005-bade6780-7477-11eb-8e9c-3ff0786b28c1.JPG)

Sorry, sorry... (and thank you very much). 

Julia

**Comment by Albert-Alvarez** [comment, marked as answer]
Have you indicated a message during the commit?

```bash
git commit -m "Your message"
```

What is shown in your terminal is the common form to indicate a message when no message is indicated in the commit command.

**Comment by ghost** [comment]
I didn't! I thought maybe the message wasn't necessary here. My mistake. 

Everything is uploaded now. Thank you for your time!
